lib/game_box_web/live/arena_live.ex

```diff
@@ -8,6 +8,9 @@ defmodule GameBoxWeb.ArenaLive do
   alias Phoenix.PubSub
 
   def render(assigns) do
+    # NOTE: don't put this in the heex template or it will be cached
```

Comment:
I know phoenix wants us to put the socket variables in the heex template. It must use these as dependencies to prevent rebroadcasting if the variables didn't change. but we have to break the rules here because the state exists in the game. we could maybe find a way around this but we need to render do this until then.